authn/github.index.js

```diff
@@ -74,12 +74,18 @@ function mainProcess(event, context, callback) {
                             { },
                             config.PRIVATE_KEY.trim(),
                             {
-                              audience: headers.host[0].value,
-                              subject: auth.getSubject(username),
-                              expiresIn: config.SESSION_DURATION,
-                              algorithm: 'RS256'
+                              "audience": headers.host[0].value,
+                              "subject": auth.getSubject(decodedData),
+                              "expiresIn": config.SESSION_DURATION,
+                              "algorithm": "RS256"
```

Comment:
How'd this change happen? (The quotes in keys.)

Reply:
by the power of copy/paste. Looks like I botched the username and decodedData variables too